lib/ansible/modules/cloud/openstack/os_ironic.py

```diff
@@ -93,6 +93,18 @@
           description:
             - size of first storage device in this machine (typically /dev/sda), in GB
           default: 1
+        capabilities:
```

Comment:
Will need `version_added: "2.8"` for these two new options.

Reply:
If this can be added to the PR, then I'm happy to get it merged.